papers/bradley_dice/55_bradley_dice.rst

```diff
@@ -0,0 +1,553 @@
+
+..
+    TODO: discuss independence from the rest of signac, possibility of releasing as a separate package?
+
+Related Software
+~~~~~~~~~~~~~~~~
+
+Are there other packages with related purposes? Vyas is not aware of any, the closest thing is Zict, a project Bradley pointed out a while ago.
+However, its scope is limited to composing mutable mappings.
+However, one natural question I'd expect from people is how this package to add a collection-like interface to some object compares to those objects directly implementing the interface.
+For example, I'd expect our closest comparison for a Redis-backed dict to be pyredis itself, which offers a dictionary-like API.
+I'd expect us to pretty much always be slower, but also to be much easier to work with and to support a lot more out-of-the-box (e.g.
+nested objects, buffering, and composition of data structures that may require more internal plumbing otherwise).
+
+
+Project Evolution
+-----------------
+
+The signac project has evolved from being an open-source project mostly developed and managed by the Glotzer Group at the University of Michigan, to being supported by over 30 contributors and committers/maintainers on 3 continents and with over 55 citations from academic and government research labs and 12 talks at large scientific, Python, and data science conferences.
+The growth in involvement with signac is the result of our focus on developing features based on user needs, as well as our efforts to transition signac users to signac contributors, through many initiatives in the past few years.
+Through encouraging users to become contributors, we ensure that signac addresses real users' needs.
+We have expanded signac's contributor involvement to outside of the University of Michigan through expanded use in diverse research groups (and through maintainers graduating and staying involved?), but more notably through the Google Summer of Code (GSoC) program.
+Our experience from the GSoC led to a new committer (explained later in this section) and much work on some of the developments presented above, namely synced collections and aggregation.
+To encourage code contributions from existing users, we maintain active support and discussion through Slack.
+In addition, we have started hosting weekly "office hours" for in-person (virtual) introduction and contributions to the code base.
+By pairing new contributors with experienced signac developers, we significantly reduce the knowledge barrier to joining a new project.
+Office hours creating space for users to make contributions has also led to more features and documentation born directly out of user need.
+Contributing to documentation has been a productive starting point for new users-turned-contributors, both for the users and the project, since it improves the users' familiarity with the API as well as addresses weak spots in the documentation more obvious to newer users.
```

Comment:
```suggestion
Contributing to documentation has been a productive starting point for new users-turned-contributors, both for the users and the project, since it improves the users' familiarity with the API as well as addresses weak spots in the documentation that are more obvious to new users.
```